Sources/ArcGISToolkit/Components/AR/FlyoverSceneView.swift

```diff
@@ -61,9 +118,20 @@ public struct FlyoverSceneView: View {
                         orientation: interfaceOrientation
                     )
                 }
+                .onChange(of: initialCamera) { initialCamera in
+                    cameraController.originCamera = initialCamera
+                }
+                .onChange(of: translationFactor) { translationFactor in
+                    cameraController.translationFactor = translationFactor
+                }
                 .observingInterfaceOrientation($interfaceOrientation)
         }
     }
+    
+    func updateCameraController() {
+        cameraController = TransformationMatrixCameraController(originCamera: initialCamera)
+        cameraController.translationFactor = translationFactor
+    }
```

Comment:
Is this method needed or used anywhere? It seems `.onChange(of: initialCamera)` and `.onChange(of: translationFactor)` update those properties already.

Reply:
no, it's not needed. I left it in by accident.